crypto/kyber/p_kyber_test.cc

```diff
@@ -143,9 +141,148 @@ TEST(Kyber512Test, EVP_PKEY_size) {
   EXPECT_TRUE(EVP_PKEY_keygen_init(kyber_pkey_ctx));
   EXPECT_TRUE(EVP_PKEY_keygen(kyber_pkey_ctx, &kyber_pkey));
 
-  EXPECT_EQ(KYBER512_PUBLICKEY_BYTES + KYBER512_SECRETKEY_BYTES, EVP_PKEY_size(kyber_pkey));
-  EXPECT_EQ(8*(KYBER512_PUBLICKEY_BYTES + KYBER512_SECRETKEY_BYTES), EVP_PKEY_bits(kyber_pkey));
+  EXPECT_EQ(KYBER512_PUBLIC_KEY_BYTES + KYBER512_PRIVATE_KEY_BYTES, EVP_PKEY_size(kyber_pkey));
+  EXPECT_EQ(8*(KYBER512_PUBLIC_KEY_BYTES + KYBER512_PRIVATE_KEY_BYTES), EVP_PKEY_bits(kyber_pkey));
+
+  EVP_PKEY_CTX_free(kyber_pkey_ctx);
+}
+
+TEST(Kyber512Test, EVP_KEM_operations_test) {
```

Comment:
nit: tests in aws-lc are named in camel case style rather than using underscores

Reply:
Sure, updated.